pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -71,7 +77,12 @@ buildPythonPackage rec {
     zlib
   ];
 
-  propagatedBuildInputs = [ filelock ];
+  propagatedBuildInputs = [
+    filelock
+    # openai-triton uses setuptools at runtime:
+    # https://github.com/NixOS/nixpkgs/pull/286763/files#r1480392652
```

Comment:
Hmm github just drops me on the files tab without opening any relevant threads 😩

Reply:
I guess, 
https://github.com/NixOS/nixpkgs/pull/286763/#discussion_r1480392652
would be a better link?

I have no idea how those references work

Reply:
Try this https://github.com/openai/triton/blob/8f47bdcc9257db82c3040539f55186f126a64edf/python/triton/common/build.py#L94-L103

Reply:
I think stacktrace would be more useful here, because they have already updated some of the surrounding parts of code in `main` branch.

Reply:
I have set link to `https://github.com/NixOS/nixpkgs/pull/286763/#discussion_r1480392652` in last rebase.

Reply:
Works for me